gallery/src/GalleryNav/GalleryNavIcons.tsx

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+export const GalleryNavIconCSS3 = () => (
+  <svg className="gallery-nav-icon gallery-nav-icon--css3"
```

Comment:
Should have `.nx-icon`.  Then the custom margin CSS that you've applied to it wouldn't be necessary

Reply:
> Should have `.nx-icon`. Then the custom margin CSS that you've applied to it wouldn't be necessary

https://github.com/sonatype/sonatype-react-shared-components/pull/596/commits/ba7aa03561a3a221e972462dee6e3a2420d53064